src/OpalCompiler-Tests/OCCodeReparatorTest.class.st

```diff
@@ -14,10 +14,10 @@ OCCodeReparatorTest >> testDeclareClassVar [
 	goo := MockForCompilation classVariableNamed: #goo ifAbsent: [ nil ].
 	goo ifNotNil: [ MockForCompilation removeClassVariable: goo ].
 
-	method := [ OpalCompiler new requestor: requestor; class: MockForCompilation ; compile: requestor text ]
+	method := [ OpalCompiler new class: MockForCompilation ; compile: requestor text ]
 		on: OCUndeclaredVariableWarning
 		do: [ :e |
-			e reparator declareClassVar.
+			e notice reparator declareClassVar.
```

Comment:
Since OCUndeclaredVariableWarning implements #reparator we could probably avoid to call #notice no?

Reply:
but I want to remove reparator in OCUndeclaredVariableWarning :(

Reply:
I removed it :p